### DISCUSSION TITLE
Action distribution?

### DISCUSSION THREAD

**Discussion by tschersach** [moonrepo/moon#1621] [Q&A] [answered]
I have the feeling that switching over to moon from nx would lead to reduced efficiency on our specific use case because we heavily distribute tasks on multiple runners. 
I feel that this feature has been in "Coming Soon" for some time and I don't want to be the annoying guy asking for a feature on an OSS project but I'm just wondering if you are still interested in releasing that feature?

Cheers :)

**Comment by milesj** [comment, marked as answer]
Yeah, that feature isn't going to happen anytime soon. With that said, how complex is your CI that you need action distribution? Would sharding be enough (which is built into moon)?

**Reply by tschersach** [reply]
I never saw that moon was providing sharding capabilities. The docs are quite terse on the feature, what is a job slice? It means splitting all potential affected jobs to run between multiple jobs?
Thanks for the answer!

**Reply by milesj** [reply]
There's some docs here: https://moonrepo.dev/docs/guides/ci#parallelizing-tasks

But basically, if you run `moon ci` in a job and it would run, let's say 100 tasks, this may be too much. So you can split it into 10 tasks across 10 jobs.
